test/hotkey.test.ts

```diff
@@ -151,3 +151,64 @@ test('allows to override hotkeys', () => {
   press(window, { code: 'KeyQ', key: 'й' })
   equal(clicked, 'bqb')
 })
+
+test('dont serching in data-keyux-ignore-hotkeys', () => {
+  let window = new JSDOM().window
+  startKeyUX(window, [hotkeyKeyUX()])
+  window.document.body.innerHTML = `
+    <ul>
+      <li data-keyux-ignore-hotkeys tabindex="0">
+        Item 1
+        <button aria-keyshortcuts="v">1</button>
+      </li>
+      <li tabindex="0">
+        Item 2
+        <button aria-keyshortcuts="v">2</button>
+      </li>
+    </ul>
+  `
+
+  let result = 0
+  let buttons = window.document.querySelectorAll('button')
+  for (let button of buttons) {
+    button.addEventListener('click', () => {
+      result += parseInt(button.textContent!)
+    })
+  }
+  press(window, { key: 'v' })
+  equal(result, 2)
```

Comment:
You can get `2` not only by clicking on button 2, but on double-clicking on button 1.

It is better to use `result = ''; result += button.textContent` for that test.